pgjdbc/src/test/java/org/postgresql/test/jdbc4/jdbc41/SchemaTest.java

```diff
@@ -189,6 +197,18 @@ private void execute(String sql) throws SQLException {
     }
   }
 
+  private void execute(String sql, Connection connection) throws SQLException {
```

Comment:
Seems like this would be better placed in TestUtil and static.

Reply:
moved

Reply:
Who should resolve it? You or me?